ydb/core/grpc_services/rpc_object_storage.cpp

```diff
@@ -144,6 +144,15 @@ bool CellFromTuple(NScheme::TTypeInfo type,
         }
         break;
     }
+    case NScheme::NTypeIds::Decimal:
+    {
+        if (tupleValue.Haslow_128()) {
```

Comment:
What if Hashigh_128() == false? Would the result of the parsing be correct? Why this place differs from the mkql_proto.cpp case?

Reply:
https://github.com/ydb-platform/ydb/blob/9335b61d3316afbf9508d435ad2fff4fed6b6b2c/ydb/public/api/protos/ydb_value.proto#L163

Here high_128 is not optional.